client/signup/jetpack-connect/index.jsx

```diff
@@ -136,8 +131,17 @@ class JetpackConnectMain extends Component {
 			this.props.jetpackConnectSite.isFetching;
 	}
 
-	getCurrentUrl() {
-		let url = this.refs.siteUrlInputRef.state.value.toLowerCase();
+	handleUrlChange = ( event ) => {
+		const url = event.target.value;
+		this.setState( {
+			currentUrl: this.cleanUrl( url ),
+			shownUrl: url,
+		} );
+		this.dismissUrl();
```

Comment:
`dismissUrl` stores in the redux tree if the user have closed the instructions screen (shown when jetpack is not active / installed), so we don't render it again when react refreshes the view.  I can't remember why this is getting called on url change ... as you said, there's no way to change the url while you are reading the instructions screen, so it shouldn't be necessary at all here.